lib/GenomeFileUtil/GenomeFileUtilImpl.py

```diff
@@ -32,6 +36,31 @@ def __init__(self, config):
         self.re_api_url = config['re-api-url']
         self.raw = config
 
+
+def _validate_catalog_param_exist(var_name):
+    """Helper function to validate catalog params exist"""
+    catalog_params = ["MAX_THREADS", "THREADS_PER_CPU"]
+    if var_name not in catalog_params:
+        raise ValueError(
+            f"currently only support those {catalog_params} catalog params"
+        )
```

Comment:
I don't really think this check is necessary since the params are hardcoded in the impl file

Reply:
Updated!

Reply:
:+1: